.flake8

```diff
@@ -16,6 +18,7 @@ ignore=F821,
   # web2py needs to compare to ``== True`` or ``== False`` for queries.
   E712
 
+# :index:`question`: **Why are these here?** I thought that moving away from web2py means we can get rid of this horrible stuff...
```

Comment:
because I just copied the file.

Reply:
:) I'll take them out. My goal is to do a code review as I add docs, so here's progress!

Merry (late) Christmas!